core/tests/protractor_desktop/embedding.js

```diff
@@ -113,6 +113,11 @@ describe('Embedding', function() {
     '\'X-Frame-Options\' to \'deny\'.',
     'chrome-error:\/\/chromewebdata\/ - Failed to load resource: the server ' +
     'responded with a status of 404 ()',
+    'chrome-error:\/\/chromewebdata\/ - Failed to load resource: the server ' +
+    'responded with a status of 400 ()',
```

Comment:
Can you please share a link where this happened and is this happening repeatedly or it happened once?

Reply:
Can you please guide which link you are asking for ? 
Initially it was not occurring but after merging from develop it happened repeatedly.

Reply:
I am asking for CI test run(on github actions) where it happened which made you to ignore the error.

Reply:
Ok, please take a look here: https://github.com/oppia/oppia/runs/3346613364#step:11:305